### PR TITLE
[4824][ADD] purchase_view_adjust

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[4824](https://www.quartile.co/web#id=4824&cids=3&menu_id=505&action=1457&model=project.task&view_type=form)

### CODE REVIEW

File: purchase_view_adjust/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This module adjusts the purchase view.
```
Comment (yostashiro) [inline purchase_view_adjust/readme/DESCRIPTION.md]:
```suggestion
This module makes adjustments to purchase related views.
```

**Review by yostashiro — APPROVED** [review]

**Review by kanda999 — APPROVED** [review]
Functional check.

**Comment by kanda999** [comment]
@AungKoKoLin1997 

Could you please also move tag field??
2024-09-04_10h33_11

**Comment by AungKoKoLin1997** [comment]
> Could you please also move tag field??

I created new PR https://github.com/qrtl/rmm-custom/pull/143 because we should handle this in it's own module.
Additionally, the field will display before `deliver to` because it comes from `purchase_stock` and I don't want to expand dependency.

**Pull request merged**